packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -45,76 +41,211 @@ export function encodeExpr(value: string) {
  * @param value Value in hex of a signed operation
  */
 export function encodeOpHash(value: string) {
-  const blakeHash = blake.blake2b(hex2buf(value), undefined, 32);
-  return b58cencode(blakeHash, prefix.o);
+  const blakeHash = blake2b(hex2buf(value), 32);
+  return b58Encode(blakeHash, Prefix.OperationHash);
 }
 
+// /**
+//  *
+//  * @description Base58 encode a string or a Uint8Array and append a prefix to it
+//  *
+//  * @param value Value to base58 encode
+//  * @param prefix prefix to append to the encoded string
+//  */
+// export function b58cencode(value: string | Uint8Array, prefix: Uint8Array): string {
```

Comment:
Can we add a typedoc tag `@deprecated use b58Encode instead` and eventually remove it in next minor release?